searx/engines/360search.py

```diff
@@ -26,22 +32,62 @@
 
 # Base URL
 base_url = "https://www.so.com"
+cache_key = f"{base_url}-cookie"
+cookie_cache_expiration_seconds = 3600
```

Comment:
should be const (uppercase) or documented if instance admins are supposed to be able to change it